bin/export_atlas_ebeye_xml.pl

```diff
@@ -703,7 +731,19 @@ sub get_and_write_experiments_info {
 	# Run the queries to create two hashes, one with info for baseline
 	# experiments and one with info for differential experiments.
     my $H_differentialExperimentsInfo = $atlasDB->fetch_differential_experiment_info_from_atlasdb( $logger );
-	my $H_baselineExperimentsInfo = $atlasDB->fetch_baseline_experiment_info_from_atlasdb( $logger );
+    my $H_baselineExperimentsInfo = $atlasDB->fetch_baseline_experiment_info_from_atlasdb( $logger );
+
+	# populate $H_differentialExperimentsInfo with experiment titles for each differential study
+	foreach my $expAcc ( keys %{ $H_differentialExperimentsInfo } ) {
+		my $title = fetch_experiment_title_from_webapi( $expAcc, $logger );
```

Comment:
Indents again

Reply:
This indent is part of the foreach loop.